### PR TITLE
termdbms: init at 2021-09-04

### PR DESCRIPTION
**Pull request by Izorkin**
###### Motivation for this change
Add termdbms version aplha.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/database/termdbms/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "termdbms";
+  version = "2021-09-04";
```
Comment (legendofmiracles) [inline pkgs/development/tools/database/termdbms/default.nix:5]: it should be unstable-date, if the program is still unreleased

File: pkgs/development/tools/database/termdbms/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  patches = [ ./viewer.patch ];
+
+  ldflags = [ "-s" "-w" "-X=main.Version=${version}" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/mathaou/termdbms/";
+    description = "A TUI for viewing and editing database files";
+    license = licenses.mit;
+    platforms = platforms.unix;
```
Comment (legendofmiracles) [inline pkgs/development/tools/database/termdbms/default.nix:24]: Set by buildGoModule

Comment (Izorkin) [reply pkgs/development/tools/database/termdbms/default.nix]: Fixed.

**Comment by mathaou** [comment]
Thanks for the consideration!

**Pull request merged**